src/Symfony/Component/Polyfill/Functions/Php70.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Polyfill\Functions;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @internal
+ */
+class Php70
+{
+    public static function random_bytes($length)
+    {
+        if (0 >= $length) {
+            // PHP 7.0 throws an \Error. E_USER_ERROR is the best we can do here.
```

Comment:
This behavior diverges from what PHP 7 does. At the very least, throw an Exception so it can be wrapped in a `try/catch` block should a developer wish to handle the failure in their own way. My worry here is that raising an `E_ERROR` takes the power out of the user's hands and risks people getting used to the way this polyfill behaves, which would add friction to adopting PHP 7.